### PR TITLE
Feat/native safe recovery module

### PR DESCRIPTION
**Pull request by JohnGuilding**
* Add native Safe module
* Make `configureRecovery` function in manager `public` instead of `internal` so it can be called for the native Safe module. This is because Safe modules do not have an equivalent to the `onInstall` function
* Add native Safe recovery module integration tests

### CODE REVIEW

File: script/DeploySafeNativeRecovery.s.sol [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import { console } from "forge-std/console.sol";
+import { SafeEmailRecoveryModule } from "src/modules/SafeEmailRecoveryModule.sol";
+
+contract DeploySafeNativeRecovery_Script is Script {
+    function run() public {
+        vm.startBroadcast(vm.envUint("PRIVATE_KEY"));
+        address manager = vm.envOr("VERIFIER", address(0));
+
+        if (manager == address(0)) {
+            manager = address(1);
```
Comment (wshino) [inline script/DeploySafeNativeRecovery.s.sol]: Is this address(1) as a placeholder to run this script?
Is it okay if we don't revert?

**Review by wshino — COMMENTED** [review]
one comment.

File: script/DeploySafeNativeRecovery.s.sol (hunk 1 above)
Comment (JohnGuilding) [reply script/DeploySafeNativeRecovery.s.sol]: Yeah this is just a placeholder, I need to update this PR to the latest changes from main

**Review by wshino — APPROVED** [review]
LGTM

**Pull request merged**